importer/yarn_to_gd.gd

```diff
@@ -143,17 +133,24 @@ static func yarn_to_gd(story: YarnStory):
 	# Start the script out with boilerplate
 	script.source_code += SCRIPT_HEADER
 
-	for thread in story.nodes:
-		# Each thread is a function
-		var body = convert_fibres(story.nodes[thread].body)
+	if (typeof(story.nodes) == TYPE_DICTIONARY):
+
+		for thread in story.nodes:
+			# Each thread is a function
 
-		script.source_code += function(thread, body)
+			var body = convert_fibres(story.nodes[thread].body)
+
+			script.source_code += function(thread, body)
+	else:
+		printerr("could not read story nodes type was {typeof(story.nodes)}")
```

Comment:
I had some kind of fatal error here when something failed to parse... I just tried to do a simple check here to make sure the nodes are Dictionary and more safely error out if there's an issue.

Reply:
Ah yes I did miss out on error handling, good catch :)